code/datums/trading/unique.dm

```diff
@@ -1,7 +1,7 @@
 /datum/trader/ship/unique
 	trade_flags = TRADER_WANTED_ONLY|TRADER_GOODS
-	want_multiplier = 5
-	typical_duration = 10
+	want_multiplier = 100
```

Comment:
That´s definitly a no

Reply:
5 times mult leads to some items from the trader being inobtaineable in hrp environment - in which case why have them at all. For example, it is perfectly reasonable to run to genetics/cloning on TG, get ten human bodies + scavenge whatever is in the hallway and then trade it for a black gygax with cannibal trader. This option doesn't exist on Aurora [nor should it] - as such to allow for at least fringe cases of buying the most expensive items the mult HAS to go up. The amount of how high up it has to go is up to debate.

I suggest anything between 20x to 100x

Reply:
I am very wary about that and the syndie trader.
With a value increase of 20-100 multiplier you could easily get some very advanced items that are otherwise only available for TCs by selling some rather worthless and easy to acquire shit to the merchant.
As the number of TCs is intentionally restricted this would give a huge buff to antagonistic merchants.
Therefore I am opposed to the multiplier increase.
-> If you want some stuff that is usually only available for TCs you will have to be creative and put some effort into it.

Reply:
Let's leave the mult default for now, we can always buff it later - will change